roles/addons/users_basic/molecule/default/prepare.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
```

Comment:
Can you use the [same layout than #175](https://github.com/bluebanquise/bluebanquise/blob/af14ffa88d7e84630c5c9ca5b0ae2d87063db29c/roles/core/display_tuning/molecule/default/prepare.yml)? (files are doing the same task, should be identical)

Reply:
done.